### PR TITLE
Tone down similarity to JavaScript.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

**Comment by LefterisJP** [comment]
C++ yes .. but how python?

**Comment by chriseth** [comment]
Inheritance is taken from Python and modifiers are inspired by python's function annotations.

**Comment by LefterisJP** [comment]
I had not thought about it but now that you mention it yes indeed the implementaiton is identical to that of python's for inheritance. Due to the syntactic difference I did not make the connection.

File: docs/index.rst [hunk 1]
```diff
@@ -7,7 +7,8 @@ Solidity
     :align: center
 
 Solidity is a contract-oriented, high-level language for implementing smart contracts.
-The syntax is similar to that of JavaScript and it is designed to target the Ethereum Virtual Machine (EVM).
+It was influenced by C++, Python and JavaScript
```
Comment (axic) [inline docs/index.rst:10]: I guess the idea for the order here is that type system came from C++, inheritance from Python and some syntax from JS?

Comment (chriseth) [reply docs/index.rst:10]: Exactly, only `var`, `function` are actually javascript.

Comment (axic) [reply docs/index.rst:10]: Some of this could be described, but it doesn't really make any difference. It is in flux anyway.

**Review by axic — APPROVED** [review]

**Pull request merged**